<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportVisitsSetupDialog</class>
 <widget class="QDialog" name="ReportVisitsSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1" colspan="2">
    <widget class="QFrame" name="frmAge_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="edtAgeFrom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeTo">
        <property name="text">
         <string>по</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtAgeTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeYears">
        <property name="text">
         <string>лет</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="chkOnlyEventExecDate">
     <property name="text">
      <string>Учитывать только даты закрытия обращения</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>Тип обращения</string>
     </property>
     <property name="buddy">
      <cstring>cmbEventType</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblAge">
     <property name="text">
      <string>Возраст с</string>
     </property>
     <property name="buddy">
      <cstring>edtAgeFrom</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblTypeFinance">
     <property name="text">
      <string>Тип финансирования</string>
     </property>
     <property name="buddy">
      <cstring>cmbTypeFinance</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbTypeFinance"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblInsurer">
     <property name="text">
      <string>СМО</string>
     </property>
     <property name="buddy">
      <cstring>cmbInsurer</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="CInsurerComboBox" name="cmbInsurer"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblTariff">
     <property name="text">
      <string>Тарификация</string>
     </property>
     <property name="buddy">
      <cstring>cmbTariff</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QComboBox" name="cmbTariff">
     <item>
      <property name="text">
       <string>не учитывать</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>тарифицированные</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>не тарифицированные</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblVisitPayStatus">
     <property name="text">
      <string>Флаг финансирования</string>
     </property>
     <property name="buddy">
      <cstring>cmbVisitPayStatus</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QComboBox" name="cmbVisitPayStatus">
     <item>
      <property name="text">
       <string>не задано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>не выставлено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>выставлено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>отказано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>оплачено</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndEventDate">
     <property name="text">
      <string>Максимальная дата ввода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndEventDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegEventDate">
     <property name="text">
      <string>Минимальная дата ввода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegEventDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="CDateEdit" name="edtEndEventDate"/>
   </item>
   <item row="0" column="2">
    <widget class="CDateEdit" name="edtBegEventDate"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CInsurerComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportVisitsSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportVisitsSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
